ldk/javascript/examples/self-test-loop/src/tests/index.ts

```diff
@@ -881,6 +882,60 @@ export const linkWhisper = (): Promise<boolean> =>
     });
   });
 
+
+export const listPairWhisperCopyable = (): Promise<boolean> =>
```

Comment:
If you wanted to get ambitious, you could use the clipboard aptitude to make sure that the text you had in the list pair was copied.

Reply:
I like that idea, I'll give that a shot.

Reply:
Did it. Good idea.